aiven/client/cli.py

```diff
@@ -3059,6 +3059,93 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List Flink tables"""
+        project_name = self.get_project()
+        self.print_response(self.client.list_flink_tables(project_name, self.args.service_name))
+
+    @arg.project
+    @arg.service_name
+    @arg("integration_id", help="Service integration ID")
+    @arg("-n", "--table-name", required=True, help="Table name")
+    @arg("--kafka-topic", required=False, help="Topic name, used as a source/sink. (Kafka integration only)")
+    @arg("--jdbc-table", required=False, help="Table name in Database, used as a source/sink. (PG integration only)")
+    @arg("-p", "--partitioned-by", required=False, help="A column from a schema, table will be partitioned by")
+    @arg(
+        "-l",
+        "--like-options",
+        required=False,
+        help="Clause can be used to create a table based on a definition of an existing table"
+    )
+    @arg("-s", "--schema-sql", required=True, help="Source/Sink table schema")
+    def service__flink__table__create(self):
+        """Create a Flink table"""
+        project_name = self.get_project()
+        self.print_response(
+            self.client.create_flink_table(
+                project_name,
+                self.args.service_name,
+                self.args.integration_id,
+                self.args.table_name,
+                self.args.schema_sql,
+                self.args.kafka_topic,
+                self.args.jdbc_table,
+                self.args.partitioned_by,
+                self.args.like_options,
+            )
+        )
+
+    @arg.project
+    @arg.service_name
+    @arg("table_id", help="Table ID")
```

Comment:
Just observation from user experience, it would be nice to be able to get table by table name, not by table id, but this might need changes with API?

Reply:
Yes. It is require a proper Metadata Table, that is in scope next month